benchmarks/routines/norm.py

```
@@ -0,0 +1,1049 @@
+            flashinfer.norm.fused_add_rmsnorm_quant(
+                out_tensor,
+                input_tensor,
+                residual_tensor,
+                weight,
+                scale=scale,
+                eps=eps,
+                enable_pdl=enable_pdl,
+            )
+            return out_tensor
+        else:
+            raise ValueError(f"Unsupported backend: {backend}")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)